### PR TITLE
Improve overfermentation when retarding a dough

### LINKED ISSUE #248: Add dough temperature to the equation in section "11.4.2  OVERFERMENTED"
I know you've written about dough temperature in other places, but it may be misleading if someone stumbles upon this section on its own. If you have high protein flour dough that's 27C and you let it double in size during bulk ferment and then retard it for 15h, chances are it will overproof as it will take longer to cool completely in the fridge. However I don't have any proof for this, it's just a theoretical suggestion :D

### PR DESCRIPTION
**Pull request by hendricius**
This adds retarding to the glossary and adds a note on how the times can be effected when starting with a warmer dough. This fixes #248

### CODE REVIEW

File: book/troubleshooting/crumb-structures.tex [hunk 1]
```diff
@@ -98,8 +98,9 @@ \subsection{Overfermented}%
 It is excellent in combination with something
 sweet or a soup.  From a consistency perspective, it is no longer as fluffy as it could be.
 The crumb might also taste a little bit gummy. That's because it has been broken down a lot
-by the bacteria. Furthermore, this style of bread has a significantly lower amount of gluten~\cite{raffaella+di+cagno}
```
Comment (hendricius) [inline book/troubleshooting/crumb-structures.tex:101]: Just saw the line was a bit long and wrapped it.

File: book/glossary/glossary.tex [hunk 2]
```diff
@@ -320,6 +320,13 @@
 shape, without the domed top characteristic of other bread loaves. Pullman loaves are
 often sliced very thin and are popular for making sandwiches.
 
+\item[\textbf{Retarding:}] The process of slowing down fermentation during the proofing
```
Comment (cedounet) [inline book/glossary/glossary.tex:323]: Please rebase on mine, I fixed several issues with the glossary.

**Review by cedounet — CHANGES_REQUESTED** [review]

File: book/troubleshooting/crumb-structures.tex [hunk 3]
```diff
@@ -138,6 +139,17 @@ \subsection{Overfermented}%
 in less acidity in your final bread. If you are a chaser of a very strong tangy
 flavor profile, then a stronger flour with more gluten will help.
 
+When retarding sourdough (cold proofing in the refrigerator), temperature plays a
+pivotal role in fermentation rates.  As the dough chills in the refrigerator,
+fermentation decelerates. Starting the retarding process at a warmer
+temperature means this deceleration takes longer.
+
+For instance, a dough that's ideal after 8 hours of retarding might be ready in
+merely 4 hours if it began at a higher temperature. Thus, it's crucial to
```
Comment (cedounet) [inline book/troubleshooting/crumb-structures.tex:148]: 4~hours

**Comment by hendricius** [comment]
@cedounet thanks! Please have another look.

**Comment by cedounet** [comment]
Lgtm

**Review by cedounet — APPROVED** [review]

**Pull request merged**